docs/orchestrating-elastic-stack-applications/agent-fleet.asciidoc

```diff
@@ -0,0 +1,402 @@
+:page_id: elastic-agent-fleet
+:agent_recipes: https://raw.githubusercontent.com/elastic/cloud-on-k8s/{eck_release_branch}/config/recipes/elastic-agent
+ifdef::env-github[]
+****
+link:https://www.elastic.co/guide/en/cloud-on-k8s/master/k8s-{page_id}.html[View this document on the Elastic website]
+****
+endif::[]
+[id="{p}-{page_id}"]
+= Run Fleet Server and Fleet-managed Elastic Agent on ECK
+
+experimental[]
+
+This section describes how to configure and deploy Elastic Agent in link:https://www.elastic.co/guide/en/fleet/current/elastic-agent-installation.html[Fleet-managed] mode with ECK.
+
+NOTE: Running Fleet Server and Fleet-managed Elastic Agent on ECK is compatible only with Stack versions 7.14+.
+
+
+NOTE: See the link:k8s-elastic-agent-standalone.html[Standalone section] if you want to run Elastic Agent in the link:https://www.elastic.co/guide/en/fleet/current/run-elastic-agent-standalone.html[standalone mode].
+
+
+
+* <<{p}-elastic-agent-fleet-quickstart,Quickstart>>
+* <<{p}-elastic-agent-fleet-configuration,Configuration>>
+
+[id="{p}-elastic-agent-fleet-quickstart"]
+== Quickstart
+
+experimental[]
+
+. Apply the following specification to deploy Fleet Server, Elastic Agents, Elasticsearch, and Kibana. ECK automatically configures secure connections between all components. Fleet will be set up and all agents will be enrolled in the default policy.
++
+[source,yaml,subs="attributes,+macros"]
+----
+cat $$<<$$EOF | kubectl apply -f -
+apiVersion: agent.k8s.elastic.co/v1alpha1
+kind: Agent
+metadata:
+  name: fleet-server
```

Comment:
For consistency with other quickstarts, should we try to use `quickstart` in resource names as suffix?